RxSwift/Disposables/CompositeDisposable.swift

```diff
@@ -102,15 +114,25 @@ public class CompositeDisposable : DisposeBase, Disposable, Cancelable {
     }
     
     /**
-    Removes and disposes the disposable identified by `disposeKey` from the CompositeDisposable.
-    
-    - parameter disposeKey: Key used to identify disposable to be removed.
-    */
+     Removes and disposes the disposable identified by `disposeKey` from the CompositeDisposable.
+     
+     - parameter disposeKey: Key used to identify disposable to be removed.
+     */
+    @available(*, deprecated, renamed: "removeDisposable(forKey:)")
     public func removeDisposable(_ disposeKey: DisposeKey) {
-        _removeDisposable(disposeKey)?.dispose()
+        removeDisposable(forKey: disposeKey)
     }
-
-    private func _removeDisposable(_ disposeKey: DisposeKey) -> Disposable? {
+    
+    /**
+     Removes and disposes the disposable identified by `disposeKey` from the CompositeDisposable.
+     
+     - parameter disposeKey: Key used to identify disposable to be removed.
+     */
+    public func removeDisposable(forKey disposeKey: DisposeKey) {
```

Comment:
I think we should have naming consistent with https://github.com/ReactiveX/RxSwift/issues/800

So probably `remove(for:)`.